sdk/core/azure-mgmt-core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.1 (2022-04-28)
+
+### Other Changes
+
+- Updated required `azure-core` version
```

Comment:
v tiny nit, don't even have to change it for this pr, but it would be nice to say the azure-core min version